Package.swift

```diff
@@ -637,7 +637,7 @@ let package = Package(
         .package(url: "https://github.com/SwiftyJSON/SwiftyJSON", .upToNextMajor(from: "5.0.2")),
         .package(
             url: "https://github.com/tuist/Rosalind",
-            .upToNextMajor(from: "0.5.108")
+            branch: "feat/type"
```

Comment:
will change to being version-based once this is merged: https://github.com/tuist/Rosalind/pull/242